mavisp/modules.py

```diff
@@ -248,6 +272,18 @@ def ingest(self, mutations):
         else:
             warnings.append(MAVISpWarningError(f"Stability classification (RaSP, FoldX) can only be calculated if exactly one RaSP and one MutateX datasets are available"))
 
+        # Add stability classification for Foldetta and FoldX-RaSP mean
+        if foldetta_mean is not None:
+            model_data[f'Stability classification (Foldetta Mean)'] = model_data.apply(self._generate_single_stability_classification, column=foldetta_mean, axis=1)
+        else:
+            warnings.append(MAVISpWarningError(f"Stability classification (Foldetta Mean) can only be calculated if Foldetta mean is available"))
```

Comment:
I think we can skip this warning - the previous warning about foldetta not being there is enough